dtool/Package.cmake

```diff
@@ -68,6 +68,9 @@ if(THIRDPARTY_DIRECTORY)
   elseif(CMAKE_SYSTEM_NAME STREQUAL "Android")
     set(_package_dir ${THIRDPARTY_DIRECTORY}/android-libs-${CMAKE_ANDROID_ARCH})
 
+  elseif(CMAKE_SYSTEM_NAME STREQUAL "WASI")
+    set(_package_dir ${THIRDPARTY_DIRECTORY})
```

Comment:
Set this to a wasi-specific subdirectory or don't support THIRDPARTY_DIRECTORY for wasi at all

Reply:
maybe THIRDPARTY_DIRECTORY should be CMAKE_INSTALL_PREFIX as default

Reply:
documenting from discord

https://cmake.org/cmake/help/latest/variable/CMAKE_PREFIX_PATH.html  could replace THIRDPARTY_DIRECTORY, and maybe default to CMAKE_INSTALL_PREFIX if not set at all.

Todo: find how cmake android-studio ( recent ndk) set/get/detect the prebuilt folder which maybe contains the per arch prebuilts from third parties.